### PR TITLE
Move help pages to component

### PR DESCRIPTION
**Pull request by nolawi**
#4412 Part 1 of 4
- [x] help component loads in react
- [x] loads component in `/help` and root
- [x] certification help component
- [x] update help spec tests

// Todo will create test for all help views in part 4

### CODE REVIEW

File: client/app/help/certificationHelp.js [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+    from VACOLS and VBMS. It will then generate a PDF of the Form 8 and allow you
+    to easily upload that document to the VBMS eFolder.
+
+    You should use Caseflow with paperless appeals that are ready to be certified.
+      </p>
+
+      <h2 id="how-to-launch">3. How do I launch Caseflow Certification?</h2>
+
+      <p>
+    Once you're ready to certify an appeal, open and log in to VACOLS then search
```
Comment (amprokop) [inline client/app/help/certificationHelp.js]: What's all this red highlighting? Do we need to escape this or use different characters?

**Comment by amprokop** [comment]
image

**Comment by amprokop** [comment]
@nolawi — i get a blank page. Did you test this PR?

**Comment by nolawi** [comment]
yes - works let me put a gif

File: client/app/containers/BaseContainer.jsx [hunk 2]
```diff
@@ -11,6 +11,7 @@ import CaseWorkerIndex from './CaseWorker/CaseWorkerIndex';
 import TestPage from './TestPage';
 import Intake from '../intake';
 import Reader from '../reader/index';
+import Help from '../help/index';
```
Comment (amprokop) [inline client/app/containers/BaseContainer.jsx]: Can we not use BaseContainer here? 

Instead

```
+<% content_for :full_page_content do %>
+  <%= react_component("Help", props: {
+    userDisplayName: current_user.display_name,
+    dropdownUrls: dropdown_urls,
+    feedbackUrl: feedback_url,
+    buildDate: build_date,
+  }) %>
+<% end %>
```

File: client/app/help/certificationHelp.js [hunk 3]
```diff
@@ -0,0 +1,330 @@
+import React from 'react';
```
Comment (amprokop) [inline client/app/help/certificationHelp.js]: All of our react component files are capitalized - `CertificationHelp.js`. Also, should this be a jsx file?

File: client/app/help/index.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+import NavigationBar from '../components/NavigationBar';
+import { COLORS } from '@department-of-veterans-affairs/appeals-frontend-toolkit/util/StyleConstants';
+import Footer from '@department-of-veterans-affairs/appeals-frontend-toolkit/components/Footer';
+import CertificationHelp from './certificationHelp';
+
+class Help extends React.PureComponent {
+
+     helpRoot = () => <div> <h1>Caseflow Help</h1>
+       <ul id="toc" className="usa-unstyled-list">
+         <li><a href="/certification/help">Certification Help</a></li>
```
Comment (amprokop) [inline client/app/help/index.js]: Can we change this to a React Router link?

File: client/app/containers/BaseContainer.jsx (hunk 2 above)
Comment (nolawi) [reply client/app/containers/BaseContainer.jsx]: 👍

File: client/app/help/certificationHelp.js (hunk 3 above)
Comment (nolawi) [reply client/app/help/certificationHelp.js]: :

File: client/app/help/index.js (hunk 4 above)
Comment (nolawi) [reply client/app/help/index.js]: helpRoot is temp, will move to out of index to Component and use routes is subsequent PR's once all the views are moved, if there is no objections.

Comment (amprokop) [reply client/app/help/index.js]: ok that works!

File: app/views/help/certification.html.erb [hunk 5]
```diff
@@ -1,360 +1,10 @@
-<% content_for :page_title do page_title("Certification Help") end %>
+<% content_for :full_page_content do %>
+  <%= react_component("Help", props: {
+    userDisplayName: current_user.display_name,
+    dropdownUrls: dropdown_urls,
+    feedbackUrl: feedback_url,
+    buildDate: build_date,
+    page: "Help"
```
Comment (amprokop) [inline app/views/help/certification.html.erb]: is this prop extraneous now?

File: app/views/help/index.html.erb [hunk 6]
```diff
@@ -1,13 +1,11 @@
...
-    <li><a href="/intake/help">Intake Help</a></li>
-  </ul>
-</div>
+<% content_for :full_page_content do %>
+  <%= react_component("Help", props: {
+    userDisplayName: current_user.display_name,
+    dropdownUrls: dropdown_urls,
+    feedbackUrl: feedback_url,
+    buildDate: build_date,
+    page: "Help"
```
Comment (amprokop) [inline app/views/help/index.html.erb]: is this prop extraneous now?

**Pull request merged**